lib/css/components/sidebar-widgets.less

```diff
@@ -0,0 +1,245 @@
+.s-sidebarwidget {
+    --_sw-content-px: calc(var(--su16) - var(--su1)); // subtract 1px for border
+    --_sw-content-spacing-inner: var(--su12); // the spacing between two adjacent simple items
+    --_sw-content-spacing-outer: var(--su16); // the spacing at the start/end of a group of simple items, as well as between a complex item and its separator line
+    --_sw-bc: var(--bc-medium);
+    --_sw-content-bc: var(--bc-light);
+    --_sw-header-bg: var(--black-025);
+
+    & &--header {
+        &.s-sidebarwidget__small-bold-text {
+            font-size: var(--fs-caption);
+            font-weight: bold;
+            .s-sidebarwidget--action {
+                font-weight: normal;
+                line-height: calc(1.3 * var(--fs-caption));  // line-height should be the same as in the outside element, so the header and action baselines line up
+            }
+        }
+
+        &.s-sidebarwidget__expanding-control {
+            cursor: pointer;
+            &:before {
+                content: '';
+                float: left;
+                margin-top: ~"calc(1.3em / 2 - 5px)"; // 1.3 is our standard line height
+                margin-right: var(--su12);
+                border: 5px solid transparent;
+                border-left-color: var(--bc-darker);
+                border-right-width: 0;
+                transition: transform 0.3s cubic-bezier(0.4, 0.4, 0.6, 1);
+            }
+            &[aria-expanded='true']:before {
+                transform: rotate(90deg);
+            }
+        }
+
+        &:first-child {
+            border-top-left-radius: var(--br-sm);
+            border-top-right-radius: var(--br-sm);
+        }
+
+        &:active {
+            outline: none;
+        }
+
+        border-top: 1px solid var(--_sw-content-bc);
+        padding: var(--_sw-content-spacing-inner) var(--_sw-content-px);
+        background: var(--_sw-header-bg);
```

Comment:
Since it's not redefined anywhere, we can just add the value of `--_sw-header-bg` (`var(--black-025)`) here and get rid of the custom property.

Reply:
Would not the same apply then also for the `--_sw-header-bc` property you suggested to introduce in your previous comment? It's only use in a single place too. https://github.com/StackExchange/Stacks/pull/1185#discussion_r1026795089

Reply:
I now understand that for `--_sw-header-bc` you want to preserve the fact that the color is inherit by `--_sw-content-bc` which you can only do with custom properties. Will address this.